src/getting_started/changelogs/Changelogs.js

```diff
@@ -15,6 +15,40 @@ export default function Authentication() {
         </p>
       </section>
       <section>
+        <h2>2018-03-14</h2>
+        <hr /><br />
+        <b>Breaking:</b><br />
+        <ul>
+          <li>Removed the ability to change passwords
+            <ul>
+              <li>Removed POST /profile/password</li>
+              <li>Removed POST /account/users/:username/password</li>
+              <li>To change your password, use the provided password reset features</li>
+            </ul>
+          </li>
+        </ul>
+        <b>Features:</b><br />
+        <ul>
+          <li>Add support for refresh tokens
```

Comment:
If you're going to say "Removed" in the first list element, you should change this to Added.

Not a huge deal; just my nitpicking.